packages/react/src/components/DataTable/__tests__/TableHeader-test.js

```diff
@@ -58,11 +60,11 @@ describe('DataTable.TableHeader', () => {
         </TableHead>
       </Table>
     );
-    expect(wrapper).toMatchSnapshot();
+    expect(container).toMatchSnapshot();
```

Comment:
Would it makes sense to test against the expected classes for some of these instead of the snapshot? that's what I've seen we've been doing so far:

```suggestion
    expect(screen.getByTestId('test-id')).toHaveClass.('cds--table-sort--active');
```

* You'd have to add a `data-testid="test-id"` to the `<TableHeader ... />` instance